syntax = "proto2";

message l2d_validate_login_request {
	required string account_name = 1;
	required uint32 server_id = 2;
}